public/background.js

```diff
@@ -1,7 +1,25 @@
 /*global chrome*/
 
 chrome.runtime.onMessage.addListener((req) => {
-  if (req == "linkwrite") {
+  if (req == "linkchat") {
```

Comment:
Why not using consts and tripple equality ? 

```suggestion
  if (req === YOUCHAT_LINKAPP) {
```